src/Process/Pool/Logger/Message/Metadata/Host/Builder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+declare(strict_types=1);
+
+namespace Neighborhoods\Kojo\Process\Pool\Logger\Message\Metadata\Host;
+
+use Neighborhoods\Kojo\Process\Pool\Logger\Message\Metadata\HostInterface;
+
+class Builder implements BuilderInterface
+{
+    use Factory\AwareTrait;
+
+    public function build() : HostInterface
+    {
+        $host = $this->getProcessPoolLoggerMessageMetadataHostFactory()->create();
+
+        $host->getHostName();
+        $host->getLoadAverage();
+
+        return $host;
```

Comment:
As with the `Metadata\Builder\Factory`, I don't think a builder is necessary here, `Metadata\Host` could be a `shared: true` actor